library/src/main/res/layout/exomedia_exo_view_layout.xml

```diff
@@ -2,9 +2,11 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                 xmlns:tools="http://schemas.android.com/tools"
                 android:layout_width="match_parent"
-              android:layout_height="match_parent">
+              android:layout_height="match_parent"
+                android:background="@android:color/black"
```

Comment:
I suppose. I was matching the existing UI, which was black with a surfaceview. Now it's white, presumably due to being transparent?